packages/grafana-ui/src/components/DateTimePickers/TimeRangeInput.tsx

```diff
@@ -28,6 +28,8 @@ export interface TimeRangeInputProps {
   hideQuickRanges?: boolean;
   disabled?: boolean;
   showIcon?: boolean;
+  /** Which day of the week the calendar should start on. Possible values: "saturday", "sunday" or "monday" */
+  weekStart?: string;
```

Comment:
We mention possible values but then just type it as `string`, is it worth it to create a type that is `WeekStart`?

Reply:
It's gonna be challenging as the type comes all the way from the API or settings. Additionally there are three places where this can come from - Org settings, User settings or dashboard settings 😄 I'll double check this just in case.

Reply:
Ok, I've added a separate type for `WeekStart`, the consumers of the component will need to make sure they provide correct types for it.